scripts/deployOldUpgradeableVersion.js

```diff
@@ -59,14 +76,85 @@ module.exports.deployOldColonyVersion = async (contractName, interfaceName, impl
     const OldInterface = contract(JSON.parse(interfaceArtifact));
     OldInterface.setProvider(web3.currentProvider);
 
-    return OldInterface;
+    const oldAuthorityArtifact = fs.readFileSync(`./colonyNetwork-${versionTag}/build/contracts/ColonyAuthority.json`);
+    const OldAuthority = contract(JSON.parse(oldAuthorityArtifact));
+    OldAuthority.setProvider(web3.currentProvider);
+
+    colonyDeployed[interfaceName] = colonyDeployed[interfaceName] || {};
+    colonyDeployed[interfaceName][versionTag] = { OldInterface, OldAuthority, resolverAddress: colonyVersionResolverAddress };
+
+    return colonyDeployed[interfaceName][versionTag];
   } catch (e) {
     console.log(e);
     return process.exit(1);
   }
 };
 
-async function deployOldUpgradeableVersion(contractName, interfaceName, implementationNames, versionTag) {
+module.exports.downgradeColony = async (colonyNetwork, colony, version) => {
+  if (!colonyDeployed.IMetaColony[version]) {
+    throw new Error("Version not deployed");
+  }
+  const accounts = await web3.eth.getAccounts();
+  const editableColony = await getColonyEditable(colony, colonyNetwork);
+
+  const oldAuthority = await colonyDeployed.IMetaColony[version].OldAuthority.new(colony.address, { from: accounts[0] });
+  const owner = await oldAuthority.owner();
+  await oldAuthority.setUserRole(accounts[0], ROOT_ROLE, true, { from: owner });
+  await oldAuthority.setOwner(colony.address, { from: accounts[0] });
+  await editableColony.setStorageSlot(0, `0x${"0".repeat(24)}${oldAuthority.address.slice(2)}`);
```

Comment:
Can we add a comment explaining why downgrading a colony requires editing storage slots?